File: SMI-SDK/Java SDK/src/pt/sapo/gis/exception/TraceSDKException.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */
package pt.sapo.gis.exception;

/**
 * @author GIS Team
 *
 * Base class for trace SDK aplication exceptions
 */
public abstract class TraceSDKException extends Exception {

    public TraceSDKException() {
        super();
    }

    public TraceSDKException(String message) {
        super(message);
    }

    public TraceSDKException(Throwable cause) {
        super(cause);
    }

    public TraceSDKException(String message, Throwable cause) {
        super(message, cause);
    }    
}

File: SMI-SDK/.NET SDK/Library/gis/exception/TraceSDKException.cs
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */
using System;
namespace pt.sapo.gis.exception
{
    /**
     * @author GIS Team
     *
     * Base class for trace SDK aplication exceptions
     */
    public abstract class TraceException : Exception
    {

        public TraceException() { }

        public TraceException(String message) : base(message) { }

        public TraceException(String message, Exception cause) : base(message, cause) { }
    }
}